todo_app/tests_e2e/test_selenium.py

```diff
@@ -12,9 +13,28 @@
 
 @pytest.fixture()
 def driver():
-    with webdriver.Firefox() as driver:
+    #with webdriver.Firefox() as driver:
+    #    yield driver
+
+    # for chromium options https://peter.sh/experiments/chromium-command-line-switches/
+    # to stop :ERROR:ssl_client_socket_impl.cc(947)] handshake failed; returned -1, SSL error code 1, net_error -101
+    # https://stackoverflow.com/questions/37883759/errorssl-client-socket-openssl-cc1158-handshake-failed-with-chromedriver-chr
+    # to stop usb error
+    #https://stackoverflow.com/questions/64927909/failed-to-read-descriptor-from-node-connection-a-device-attached-to-the-system
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    opts.add_experimental_option('excludeSwitches', ['enable-logging'])
+    opts.add_argument('--no-sandbox')
+    #opts.add_argument('--disable-dev-shm-usage')
+    #opts.add_argument('--disable-webgl')
+    #opts.add_argument('--disable-gpu')
+    #opts.add_argument('--ignore-certificate-errors')
+    #opts.add_argument('--ignore-ssl-errors')
+    #opts.add_argument('--ignore-certificate-errors-spki-list')
```

Comment:
Do you need these options or not? If you don't think you need these, then delete the commented out code.